etl/steps/data/garden/un/2023-08-02/comtrade_pandemics.meta.yml

```diff
@@ -17,130 +24,145 @@ tables:
+
       import_cif_non_surgical_gloves:
         title: Imports of vulcanized (other than hard rubber), gloves other than surgical gloves.
         short_unit: "$"
         unit: "current US$"
-        description: >-
+        description_short: >-
           Imports of non-surgical gloves.
+
       import_cif_plastics_clothing:
         title: Imports of plastics in clothing
         short_unit: "$"
         unit: "current US$"
-        description: >-
+        description_short: >-
           Imports of articles of apparel and clothing accessories (including gloves, mittens and mitts).
+
       import_cif_spectacles_goggles:
         title: Imports of spectacles, goggles and the like
         short_unit: "$"
         unit: "current US$"
-        description: >-
+        description_short: >-
           Imports of spectacles, goggles and the like; (other than sunglasses) corrective, protective or other.
+
       import_cif_surgical_gloves:
         title: Imports of surgical gloves
         short_unit: "$"
         unit: "current US$"
-        description: >-
+        description_short: >-
           Imports of vulcanized (other than hard rubber), surgical gloves.
+
       import_cif_textiles_made_up_articles:
         title: Imports of textiles and made up articles
         short_unit: "$"
         unit: "current US$"
-        description: >-
+        description_short: >-
           Imports of textiles. This includes made up articles (including dress patterns).
+
       import_cif_total_pandemics:
         title: Imports of pandemic-related customs
         short_unit: "$"
         unit: "current US$"
-        description: >-
-          Total imports from any of the following products:
-
-            - Breathing appliances and gas masks
-            - Gloves, mittens and mitts (knitted or crocheted)
-            - Gloves, mittens and mitts (not knitted or crocheted)
-            - Vulcanized (other than hard rubber), gloves other than surgical gloves.
-            - Plastics in clothing
-            - Spectacles, goggles and the like
-            - Surgical gloves
-            - Textiles and made up articles
-
-
-      import_cif_breathing_appliances_per_capita:
+        description_short: >-
+          Total imports of pandemic-related customs.
+        description_key:
+          - |-
+            Products included are: "Breathing appliances and gas masks", "Gloves, mittens and mitts (knitted or crocheted)", "Gloves, mittens and mitts (not knitted or crocheted)",
+            "Vulcanized (other than hard rubber), gloves other than surgical gloves.",  "Plastics in clothing",  "Spectacles, goggles and the like",  "Surgical gloves", and
+            "Textiles and made up articles".
+      ort_cif_breathing_appliances_per_capita:
         title: Imports of breathing appliances and gas masks (per capita)
         unit: "current US$ per capita"
-        description: >-
+        description_short: |-
```

Comment:
I just noticed you use "|-" here but then have a line break in the middle of a sentence. I suppose you should use ">-" instead.